### ISSUE TITLE
Error installing from luarocks

### ISSUE THREAD

**Issue by martin-damien** [martin-damien/babel#14]
![capture d ecran 2015-11-24 a 08 47 09](https://cloud.githubusercontent.com/assets/295478/11360809/08186c9e-9288-11e5-9c72-6808a8a151e7.png)

**Comment by martin-damien** [comment]
Install randomly crash on language files:

```bash
22:28 ≡ luarocks install --local babel                                                       ᴦ/p/ luarocks ≡ MASTER
Installing https://luarocks.org/babel-1.2-1.src.rock...
Using https://luarocks.org/babel-1.2-1.src.rock... switching to 'build' mode
stat: malsukcesis eltrovi statinformon pri «locales/se-SE.lua»: Dosiero aŭ dosierujo ne ekzistas

Error: Build error: Failed installing locales/se-SE.lua in /home/damien/.luarocks/lib/luarocks/rocks/babel/1.2-1/lua/locales/se-SE.lua: locales/se-SE.lua: No such file or directory
22:29 ≡ luarocks install --local babel                                                       ᴦ/p/ luarocks ≡ MASTER
Installing https://luarocks.org/babel-1.2-1.src.rock...
Using https://luarocks.org/babel-1.2-1.src.rock... switching to 'build' mode
stat: malsukcesis eltrovi statinformon pri «locales/eo-EO.lua»: Dosiero aŭ dosierujo ne ekzistas

Error: Build error: Failed installing locales/eo-EO.lua in /home/damien/.luarocks/lib/luarocks/rocks/babel/1.2-1/lua/locales/eo-EO.lua: locales/eo-EO.lua: No such file or directory
22:29 ≡ luarocks install --local babel                                                       ᴦ/p/ luarocks ≡ MASTER
Installing https://luarocks.org/babel-1.2-1.src.rock...
Using https://luarocks.org/babel-1.2-1.src.rock... switching to 'build' mode
stat: malsukcesis eltrovi statinformon pri «locales/ca-ES.lua»: Dosiero aŭ dosierujo ne ekzistas

Error: Build error: Failed installing locales/ca-ES.lua in /home/damien/.luarocks/lib/luarocks/rocks/babel/1.2-1/lua/locales/ca-ES.lua: locales/ca-ES.lua: No such file or directory
```

**Comment by martin-damien** [comment]
Fixed in ab78c01f3b8539ba6de1141a82a40c6f77b89904 thanks to http://stackoverflow.com/questions/36729406/why-have-i-errors-when-installing-from-luarocks .
